src/StatsdClient/Statsd.cs

```diff
@@ -50,7 +50,22 @@ public abstract class Metric : ICommandType
             }
         }
 
-        public class Event : ICommandType
+        public abstract class Command : ICommandType
+        {
+            protected static string EscapeContent(string content)
+            {
+                return content
+                    .Replace("\r", "")
+                    .Replace("\n", "\\n");
+            }
+
+            protected static string TruncateOverage(string str, int overage)
+            {
+                return str.Substring(0, str.Length - overage);
+            }
+        }
+

```

Comment:
> Are you saying it is odd that Metric doesn't inherit the new Command class?
Yes.  :)

> I did feel this was a little yuck, those methods could be placed anywhere else to make it cleaner.
I wonder if we could simply move these methods to `StatsD` class instead.